src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/UpdateSubmission.java

```diff
@@ -53,6 +57,15 @@ public class UpdateSubmission {
     @JsonProperty(BENEFICIAL_OWNERS_FIELD)
     private BeneficialOwnersStatementType beneficialOwnerStatement;
 
+    @JsonProperty(CHANGE_BO_RELEVANT_PERIOD)
+    private RelevantStatementsType changeBORelevantPeriod;
+
+    @JsonProperty(TRUSTEE_INVOLVED_RELEVANT_PERIOD)
+    private RelevantStatementsType trusteeInvolvedRelevantPeriod;
+
+    @JsonProperty(CHANGE_BENEFICIARY_RELEVANT_PERIOD)
```

Comment:
changes to the model classes will need to be added to the /spec/schema.json file as well

Reply:
ignore this, i looked and saw that they have been previously added